modules/magento-ami/scripts/ec2_install/scripts/install_mage_credentials.py

```diff
@@ -17,26 +18,29 @@
   ssm_path_prefix = ssm_config['ssm_path_prefix']
 
 try:
-  mage_composer_username = ssm.get_parameter(f"{ssm_path_prefix}mage_composer_username", WithDecryption=True)
-except:
-  print("mage_composer_username: NotFound")
+    mage_composer_username = ssm.get_parameter(
+        Name=f"{ssm_path_prefix}mage_composer_username", WithDecryption=True)
+except Exception as e:
+    print(
+        f"mage_composer_username: NotFound {ssm_path_prefix}mage_composer_username", e)
```

Comment:
i dont like when editors do this formatting

```suggestion
    mage_composer_username = ssm.get_parameter(
       Name=f"{ssm_path_prefix}mage_composer_username", 
       WithDecryption=True
     )
except Exception as e:
    print(f"mage_composer_username: NotFound {ssm_path_prefix}mage_composer_username: {e}")
```